src/org/zaproxy/zap/extension/ascanrulesAlpha/ZapAddOn.xml

```diff
@@ -1,6 +1,6 @@
 <zapaddon>
 	<name>Active scanner rules (alpha)</name>
-	<version>21</version>
+	<version>25</version>

```

Comment:
Manifest version should still be 21, it seems like 20 is the current alpha release:
https://github.com/zaproxy/zap-extensions/releases

Reply:
21 has been released, you'll have to check when you get to dealing with this and set it properly.